tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -95,7 +95,9 @@ typedef enum {
   TEST_TYPE_FLEX_TENSOR_1, /**< pipeline for flexible tensor (sink) */
   TEST_TYPE_FLEX_TENSOR_2, /**< pipeline for flexible tensor (converter, static multi-tensors to flex) */
   TEST_TYPE_FLEX_TENSOR_3, /**< pipeline for flexible tensor (converter, flex to static) */
-  TEST_TYPE_TENSORS, /**< pipeline for tensors with tensor_mux */
+  TEST_TYPE_TENSORS_MUX_1, /**< pipeline for tensors with tensor_mux */
+  TEST_TYPE_TENSORS_MUX_2, /**< pipeline for tensors with tensor_mux */
```

Comment:
Could you add extra comments how this is different from `TEST_TYPE_TENSORS_MUX_1`

Reply:
updated